From 31dcbd5bccda4e0a195c0d3fec49c7fcea97f4ec Mon Sep 17 00:00:00 2001
From: Silvio Traversaro <silvio.traversaro@iit.it>
Date: Sun, 21 Mar 2021 19:05:11 +0100
Subject: [PATCH] Update CImageLoaderJPG.cpp

---
 source/Irrlicht/CImageLoaderJPG.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/source/Irrlicht/CImageLoaderJPG.cpp b/source/Irrlicht/CImageLoaderJPG.cpp
index 6b51bd98d..efa8725ce 100644
--- a/source/Irrlicht/CImageLoaderJPG.cpp
+++ b/source/Irrlicht/CImageLoaderJPG.cpp
@@ -68,7 +68,7 @@ void CImageLoaderJPG::init_source (j_decompress_ptr cinfo)
 boolean CImageLoaderJPG::fill_input_buffer (j_decompress_ptr cinfo)
 {
 	// DO NOTHING
-	return TRUE;
+	return (boolean)TRUE;
 }
 
 
